research/src/pages/select.mdx

```diff
@@ -0,0 +1,132 @@
+---
+menu: Components
+name: Select (Working Draft)
+path: /components/select
+research: /components/select.research
+---
+
+import SelectAnatomy from '../components/select-anatomy'
+
+The `<select>` element is a control that allows the user to select from a set of potential options.
+
+## Anatomy
+
+### Diagram
+
+<SelectAnatomy />
+
+### Part Definitions
+
+`::host` — Root element that contains the button and the pop-up.
+
+`::part-button` — An interactive element that toggles the appearance of the pop-up. This element has a tab stop.
+
+`::part-pop-up` — An interactive element containing the set of options the user can select from.
+
+### Events
+
+- `::host` Listening for `toggle-pop-up` event to hide/show `::part-pop-up`
+
+- `::part-button` Listens to `click` event and dispatches `toggle-pop-up` event
+
+- `::part-button` Listening for `option-change` event to update selected value shown
+
+- `::part-pop-up` Listening for `option-change` event to dispatch `toggle-pop-up` event
+
+- `::part-button` Dispatches custom event `split-button-clicked`
```

Comment:
Split button or select?

Reply:
This could be confusion on my part; I wouldn't equate the button here to a split button as it's a single action with the sole intent of toggling the pop-up (whether on keydown of enter/space or click)

Reply:
agreed, as noted above I'd expect this to be a button or a new element/sub pieces that still live within a slot that allow the content to be replaced (eg: be able to build a combo box from the select)